src/probnum/quad/solvers/policies/__init__.py

```diff
@@ -1 +1,2 @@
-from ._policy import Policy, RandomPolicy
+from ._policy import Policy
```

Comment:
```suggestion
"""Policies / Acquisition functions for Bayesian quadrature."""

from ._policy import Policy
```

Reply:
changed it to "Policies for Bayesian quadrature." as this module does not contain acquisition functions.